analyzer/src/main/kotlin/managers/Bundler.kt

```diff
@@ -219,7 +219,7 @@ class Bundler(name: String, analyzerConfig: AnalyzerConfiguration, repoConfig: R
         run(workingDir, "install", "--path", "vendor/bundle")
     }
 
-    private fun queryRubygems(name: String, version: String): GemSpec? {
+    private fun queryRubygems(name: String, version: String, retryCount: Int = 3): GemSpec? {
```

Comment:
Commit message nit: "OkHttp has a"
`retriedCount` sounds like a counter how often it was already retried. Maybe just `retries`?

Reply:
Actually is already says "retryCount", not "retriedCount". Is that good enough?